### DISCUSSION TITLE
제가 이해한것이 맞는건지..

### DISCUSSION THREAD

**Discussion by kth0833** [changdoc/naver-vaccine-macro#160] [General]
제가 이해한 바로는, [우리동네백신알림 - 예약신청] 에서 자기 동네의 병원 하나를 선택하고 그 신청페이지를 미리 띄워놓고 자동신청 하는 프로세스 인것 같은데 무조건 한 병원만 노려야되는거 맞죠?

7. 예약 신청 페이지 내에 생성된 '자동 예약 시작' 버튼을 누르면 동작이 시작됩니다.
여러 페이지를 띄워놓고 수행한다면 한 곳에서만 '자동 예약 시작' 버튼을 누르고 나머지 페이지에서는 브라우저에서 새로 고침 하면 동작됩니다.

이거보면 여러병원 한번에 노릴 수 있는 방법이 있는건가.. 해서 여쭤봅니다.

**Comment by changdoc** [comment]
초기에 작성한 가이드라 여러 병원을 띄울때 동작방식 설명이 남아있네요
#42 내용 보시면 네이버측에서 차단하게 되는 상황 설명 보실 수 있습니다
